src/zc/table/fieldcolumn.py

```diff
@@ -177,12 +182,12 @@ def renderWidget(self, item, formatter, **kwargs):
             'value=%s' % quoteattr(lbl)]
         return '<%s />' % (' '.join(res))
 
-    ###### customization API (expected to be overridden) ######
+    # customization API (expected to be overridden) ######
 
     def getLabel(self, item, formatter):
-        return super(SubmitColumn, self).renderHeader(formatter) # title
+        return super(SubmitColumn, self).renderHeader(formatter)  # title
 
-    ###### basic API ######
+    # basic API ######
```

Comment:
If you're removing the `#####` decoration from the front of the comment, remove it from the back as we..